app/views/handlers/list_remove_question.py

```diff
@@ -32,6 +40,8 @@ def handle_post(self):
                 self.parent_block["for_list"]
             )
 
+        self.questionnaire_store_updater.update_progress_for_dependent_sections()
```

Comment:
Is this needed on top of the existing `evaluate_and_update_section_status_on_list_change`?. If so why?. In add and remove the order of them is also different. Are we not reevaluating the same sections in the two methods, or at least a sub set of the are duplicates?.

Reply:
Related comment https://github.com/ONSdigital/eq-questionnaire-runner/pull/1079#discussion_r1180014934

Reply:
OK, updated it all as suggested, can't find method name suggestion so I tried to be creative.